src/LayerPlan.cpp

```diff
@@ -135,49 +135,11 @@ ExtruderTrain* LayerPlan::getLastPlannedExtruderTrain()
     return last_planned_extruder;
 }
 
-bool LayerPlan::CombBoundaryRequired()
-{
-    const CombingMode combing_mode = Application::getInstance().current_slice->scene.current_mesh_group->settings.get<CombingMode>("retraction_combing");
-    if (combing_mode == CombingMode::OFF || (layer_nr < 0 && combing_mode == CombingMode::NO_SKIN))
-    {
-        return false;
-    }
-    return true;
-}
-
-Polygons LayerPlan::computeMinimumCombBoundary()
+Polygons LayerPlan::computeCombBoundary(const bool minimumBoundary)
```

Comment:
Rename the variable to not use camel case.

```suggestion
Polygons LayerPlan::computeCombBoundary(const bool minimum_boundary)
```